liquibase-couchbase/src/test/java/system/change/RemoveDocumentsSystemTest.java

```diff
@@ -48,13 +68,28 @@ void cleanUpd() {
         if (collectionOperator.docExists(docId2)) {
             collectionOperator.removeDoc(doc2);
         }
+        if (collectionOperator.collectionIndexExists(INDEX)) {
+            collectionOperator.dropIndex(INDEX);
+        }
```

Comment:
We can drop index after all tests. No need to create and drop it every time

Reply:
Done, thanks!